catalog/security/catalog-security-policyplugin/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -18,22 +18,22 @@
                                update-strategy="container-managed" />
         <property name="createPermissions">
             <array>
-                <value>http://schemas.xmlsoap.org/ws/2005/05/identity/claims/role=guest</value>
+                <value>""</value>
```

Comment:
I haven't looked at this in a while but does this mean that by default no one will be able create/update/delete?

Reply:
It means exactly the opposite. Also, I don't think the "" is what we necessarily want here. I think we can just leave these properties completely unset.

Reply:
Nope, this means there are no restrictions as to who can perform CRUD operations.

Reply:
Ah Scott beat me to it. And also okay, I can just remove the string altogether.

Reply:
Should we add a comment to clarify that?

Reply:
Added comments to the blueprints to explain the empty values.